pom.xml

```diff
@@ -34,24 +34,13 @@
   </parent>
```

Comment:
or just remember to run `mvn org.apache.maven.plugins:maven-release-plugin:2.5.2:prepare org.apache.maven.plugins:maven-release-plugin:2.5.2:perform -B`